README.md

```diff
@@ -0,0 +1,136 @@
+[![Slack Status](http://slack.sparkpost.com/badge.svg)](http://slack.sparkpost.com)
+
+This is a SendGrid to SparkPost email template migration tool.
+
+The tool presents a simple web UI for migrating templates directly from your SendGrid account to SparkPost as well as translating SendGrid template text directly into its SparkPost equivalent.
+
+If you want to automate your migration, the tool also has an API.
+
+See below for details on [deployment on Heroku](#heroku-deployment), [manual deployment](#manual-deployment) and [API use](#the-api).
+
+## Supported Features
+- <%subject%>, <%body%>
+- Sender Info
+- [Weblink]
+- [%email%]
+- [%custom_field%], [%custom_field | default value%]
+
+## Unsupported Features
+- <%asm_global_unsubscribe_url%>
+- [Unsubscribe]
+- _User delimited variables_
+You need provide the names of the custom delimited variables during translation. Only one type of delimiter is supported. 
```

Comment:
This is old text (before our review, i'll fix it to add delimiter, not variable). And will add that `to`.